src/main/java/uk/gov/hmcts/reform/authorisation/healthcheck/ServiceAuthHealthIndicator.java

```diff
@@ -5,10 +5,12 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.actuate.health.Health;
 import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationHealthApi;
 
 @Component
+@ConditionalOnProperty(prefix = "idam.s2s-auth", name = "enabled", matchIfMissing = true)
```

Comment:
the other libraries have been set to use the url probably not having the value `false`

Reply:
https://github.com/hmcts/document-management-client/pull/8/files

Reply:
although it is illogical to have `url=false` in config in order to use as conditional but applied the change regardless

Reply:
the reason that exists is because of how we can't conditionally add variables per environment but we can change their values, not a great solution but its what was done.

Reply:
fur enough 🐈